modules/keycloak/images.yml

```diff
@@ -22,11 +23,11 @@ images:
       context: custom/keycloak
       args:
         - name: KEYCLOAK
-          value: "22.0.1"
+          value: "22.0.5"
         - name: KEYCLOAK_METRICS_SPI
           value: "4.0.0"
     tag:
-      - "22.0.1"
+      - "22.0.5"
```

Comment:
Are we removing the old one ?

Reply:
No, but if I leave the old tag, wouldn't it be overwritten with the new custom image?

Reply:
For now this is ok, but you can use the "tag" as the argument for the base image tag